kubernetes-deployment/test/test_maintainlib_common.py

```diff
@@ -119,6 +119,40 @@ def test_package(self):
 
 
 
+    def test_ipv4_address_validation_correct(self):
+
+        addr = "128.0.0.1"
+        self.assertTrue(common.ipv4_address_validation(addr))
+
+
+
+    def test_ipv4_address_validation_wrong(self):
+
+        addr1 = "128.0.0.x"
```

Comment:
>128.0.0.x [](start = 17, length = 9)

please merge the valid and invalid tests into one.
also, please test 
1) ipv6 address, e.g., 0x:ef:ff:....
2) localhost
3) 127.0.0.1 (we should consider this as invalid, right?)
4) string, e.g., "mydefaultip"
5) invalid value in each ., e.g., 0.-1.0.0, 0.0.358.0

Reply:
127.0.0.1 is valid. other are all invalid.

Reply:
256.0.0.0 will cover the scenario of 0.0.358. (0  -  255)